includes/handler/class-reject.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * Reject handler file.
+ *
+ * @package Activitypub
+ */
+
+namespace Activitypub\Handler;
+
+use Activitypub\Notification;
+use Activitypub\Collection\Actors;
+use Activitypub\Collection\Following;
+use Activitypub\Collection\Outbox;
+
+use function Activitypub\object_to_uri;
+
+/**
+ * Handle "Reject" requests.
+ */
+class Reject {
+	/**
+	 * Initialize the class, registering WordPress hooks.
+	 */
+	public static function init() {
+		\add_action(
+			'activitypub_inbox_reject',
+			array( self::class, 'handle_reject' ),
+			10,
+			2
+		);
+
+		\add_filter(
+			'activitypub_validate_object',
+			array( self::class, 'validate_object' ),
+			10,
+			3
+		);
+	}
+
+	/**
+	 * Handles "Reject" requests.
+	 *
+	 * @param array $reject  The activity-object.
+	 * @param int   $user_id The id of the local blog-user.
+	 */
+	public static function handle_reject( $reject, $user_id ) {
+		// Validate that there is a Follow Activity.
```

Comment:
```suggestion
		// Validate that there is a preceding Activity.
```
To make it more generally applicable